doc/jvm/xgboost4j_spark_tutorial.rst

```diff
@@ -153,6 +153,39 @@ Now, we have a DataFrame containing only two columns, "features" which contains
 "sepal length", "sepal width", "petal length" and "petal width" and "classIndex" which has Double-typed
 labels. A DataFrame like this (containing vector-represented features and numeric labels) can be fed to XGBoost4J-Spark's training engine directly.
 
+Dealing with missing values:
```

Comment:
@hcho3 is it the right way to make it as a subsection?

Reply:
You are right, fixed again before pull req. @CodingCat